bin/pg_repack.c

```diff
@@ -600,13 +628,27 @@ is_requested_relation_exists(char *errbuf, size_t errsize){
 		if (iparam < num_relations)
 			appendStringInfoChar(&sql, ',');
 	}
+	for (cell = exclude_table_list.head; cell; cell = cell->next)
```

Comment:
This logic is probably unnecessary since `exclude-table` conflicts with `table` and `parent-table`:
https://github.com/reorg/pg_repack/pull/360/files#diff-a9d4d5b8968d339c50f089a6b531b1f36eedd27cdc008bdb26b70fb9fd0eee2fR377